apps/wallet/src/ui/app/pages/home/tokens/coin-balance/index.tsx

```diff
@@ -24,10 +30,7 @@ function WalletBalanceUsd({ amount: walletBalance }: WalletBalanceUsdProps) {
     const walletBalanceInUsd = useMemo(() => {
         if (!formattedWalletBalance) return null;
 
-        return `~${formattedWalletBalance.toLocaleString('en', {
-            style: 'currency',
-            currency: 'USD',
-        })} USD`;
+        return `~${formatBalanceToUSD(formattedWalletBalance)} USD`;
```

Comment:
I was already working on a [task](https://github.com/iotaledger/iota/issues/3679) for this :face_with_peeking_eye: but since you have it now, if you want, you can wrap up both tasks... I’d just remove the condition in the wallet that shows it only on mainnet :pray:

Reply:
sorry i didn't see it🙏🏽